pyproject.toml

```diff
@@ -9,8 +9,12 @@ packages = [{include = "rosa"}]
 [tool.poetry.dependencies]
 python = "^3.8"
 python-simple-logger = "*"
+redhat-qe-cloud-tools = "^1.0.6"
 
 
+[tool.poetry.group.dev.dependencies]
+ipdb = "^0.13.13"
```

Comment:
Leftover?

Reply:
No, it's under dev section
This is for local development, so we will have `ipdb` for debugging.